slack-clipper-backend/app.py

```diff
@@ -36,38 +37,50 @@ def get_conversation_thread(client, channel, ts):
         messages.append({
             "text": message["text"],
             "name": users[user_id]["name"],
-            "profile_picture": users[user_id]["profile_picture"],
-            "files": message["files"]
+            "profile_picture": users[user_id]["profile_picture"]
         })
 
+        if "files" in message:
+            permalinksToAttachments = ""
+            for file in message["files"]:
+                if not file['public_url_shared']:
+                    app.client.files_sharedPublicURL(file=file["id"], token=os.environ.get("USER_TOKEN"))
+                permalinksToAttachments += '<a href="'+file["permalink_public"]+'">Download</a><div>'
```

Comment:
can we use String formatting and not concatenation

https://realpython.com/python-string-formatting/#2-new-style-string-formatting-strformat

Reply:
That will make this section a bit easier to read